devices/updateStatus.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const dynamodb = require('../lib/dynamodb');
+const utils = require('../lib/utils');
+var _ = require('underscore');
+const constants = require('../lib/constants');
+
+module.exports.updateStatus = (event, context, callback) => {
+
+  const input = JSON.parse(event.body);
+  const params = {
+    TableName: process.env.DEVICES_TABLE_NAME,
+    IndexName: 'DevicesChipIdIndex',
+    KeyConditionExpression: 'chipId = :chipId',
+    ExpressionAttributeValues: { ':chipId': input.chipId }
+  };
+
+  dynamodb.query(params, function (err, result) {
+    if (err) {
+      console.log(err);
+      callback(null, utils.createResponse(500, 'An internal server error occurred'));
+      return;
+    }
+    var attributeUpdates = utils.toAttributeUpdates({
```

Comment:
Need to check if you can find the device with given chip Id or not before updating